packages/auto-icons/src/index.ts

```diff
@@ -11,18 +11,19 @@ export default defineWxtModule<AutoIconsOptions>({
   async setup(wxt, options) {
     const parsedOptions = defu(options, {
```

Comment:
You can defu type safe with:

```suggestion
    const parsedOptions = defu<AutoIconOptioms, AutoIconOptioms>(options, {
```

Reply:
Says to prevent further typos suggests a fix with typos 😂